src/main/java/org/photonvision/PhotonCameraExtended.java

```diff
@@ -93,8 +79,8 @@ public void updateInputs(PhotonCameraExtendedInputs inputs) {
         try {
             inputs.cameraMatrix = cameraIntrinsicsSubscriber.get();
             inputs.distCoeffs = cameraDistortionSubscriber.get();
-            inputs.pipelineResult = super.getLatestResult();
-            inputs.pipelineResultTimestamp = inputs.pipelineResult.getTimestampSeconds();
+            inputs.pipelineResults = super.getAllUnreadResults().toArray(PhotonPipelineResult[]::new);
```

Comment:
Maybe later we can just pick up the "latest unread result" to solve this serialization issue if AdvantageKit doesn't solve this -- additionally, this would be a lot of data to log in any case.